lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
can you explain what you mean about constructing the power node? I think because it's a lot simpler structure than Polynomial Terms you could just build it yourself. But I know that stuff is confusing, so I can help you figure out that code and stuff if you want!